core/src/main/java/io/undertow/websockets/core/WebSocketChannel.java

```diff
@@ -50,7 +53,8 @@
  * @author Stuart Douglas
  */
 public abstract class WebSocketChannel extends AbstractFramedChannel<WebSocketChannel, StreamSourceFrameChannel, StreamSinkFrameChannel> {
-
+    public static final String WEB_SOCKETS_READ_TIMEOUT = "io.undertow.websockets.core.WEB_SOCKETS_READ_TIMEOUT";
```

Comment:
The majority of our properties are standardized in a way that we don't have upper case letters and underscore in their names.
Also, we are not mapping their names one-to-one with packages
Cana you please replace it by something like:"io.undertow.websockets.core.read-timeout". Maybe we can even subtract the core from the property name:
"io.undertow.websockets.read-timeout"